apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.cs

```diff
@@ -227,7 +227,17 @@ public PooledSession CreateDetachedSession(SessionName sessionName, ByteString t
         {
             GaxPreconditions.CheckNotNull(sessionName, nameof(sessionName));
             GaxPreconditions.CheckNotNull(transactionId, nameof(transactionId));
-            return PooledSession.FromSessionName(_detachedSessionPool, sessionName).WithTransaction(transactionId, transactionMode, readTimestamp);
+            return PooledSession.FromSessionName(_detachedSessionPool, sessionName).WithTransaction(transactionId, BuildTransactionOptions(), readTimestamp);
+
+            // TODO: Maybe just consider the breaking change and directly receive TransactionOptions.
```

Comment:
Am I right in saying detached sessions are likely to change a bit in other ways anyway?